### PR TITLE
feat (prediction) use user language for tradingview

### PR DESCRIPTION
**Pull request by ciruz**
Noticed the language in the TradingView is hardcoded so I extended it, now the language from the user settings getting used in the TradingView.

image

image

If the language isn't available, the TradingView widget will be english by default (so no issues with incompatible codes).

Some language codes from PCS are not compatible with the TradingView widget, since their language codes are also a little inconsistent at some point (german is "de_DE", spanish is "es").

image

I'm not sure about this one, maybe one day someone extend the language interface ex. for a tradingview language, to be fully compatible:

image

### CODE REVIEW

File: src/views/Predictions/components/TradingView.tsx [hunk 1]
```diff
@@ -38,19 +39,20 @@ const initializeTradingView = (TradingViewObj: any, theme: DefaultTheme) => {
 }
 
 const TradingView = () => {
+  const { currentLanguage } = useContext(LanguageContext);
```
Comment (hachiojidev) [inline src/views/Predictions/components/TradingView.tsx:42]: `const { currentLanguage } = useTranslation()`

**Review by hachiojidev — CHANGES_REQUESTED** [review]
Nice work, just one thing to adjust.

**Comment by Chef-Chungus** [comment]
@ciruz - Please run prettier on this one too

File: src/views/Predictions/components/TradingView.tsx (hunk 1 above)
Comment (ciruz) [reply src/views/Predictions/components/TradingView.tsx]: thanks

**Comment by ciruz** [comment]
done @Chef-Chungus

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**